app/lib/json_web_token.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+class JsonWebToken
+  def self.encode(payload)
+    payload.reverse_merge!(meta)
+    JWT.encode(payload, private_key, "RS256")
+  end
+
+  def self.decode(token)
+    JWT.decode(token, public_key, true, algorithm: "RS256")
+  end
+
+  # Default options to be encoded in the token
+  def self.meta
+    {
+      exp: 7.days.from_now.to_i
```

Comment:
I think we can extend this to 30 days or more

Reply:
Done!